prisma/seed.ts

```diff
@@ -0,0 +1,40 @@
+import { PrismaClient } from '@prisma/client';
+
+const prisma = new PrismaClient();
+
+async function main() {
+  const users = await prisma.user.createMany({
+    data: [
+      { email: 'bob@example.com', password: 'test', id: '1' },
```

Comment:
**issue**: Storing plain text passwords is insecure. Consider hashing passwords before storing them.